src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Windows.txt

```diff
@@ -0,0 +1,74 @@
+﻿@ECHO OFF
+SETLOCAL
+
+SET PACKAGE_DIR=%1
+SET EXECUTION_DIR=%2
+
+IF DEFINED PACKAGE_DIR ( echo Using %PACKAGE_DIR% as folder for resolving package dependencies.) ELSE (
+echo Please specify an package source directory using PACKAGEROOT parameter
+goto ShowUsage
+)
+
+if "%EXECUTION_DIR%" == "" (set EXECUTION_DIR=%~dp0)
+echo Executing in %EXECUTION_DIR% 
+
+:: ========================= BEGIN Test Execution ============================= 
+:RunTests
+
+:: Copy over dependencies... 
+GOTO CopyFiles
```

Comment:
I like leaving it this way specifically because the command is kept in the first page of the text editor, versus scrolling through every dependency before getting to it.  I can make it inline though.